File: Quickrent Dotnet/Quickrent/DTO/ProductDTO/SellerProductAddDTO.cs
namespace Quickrent.DTO.ProductDTO
{
    public class SellerProductAddDTO
    {
        public string Title { get; set; }
        public string BrandName { get; set; }
        public string ModelName { get; set; }
        public string Description { get; set; }
        public string Specifications { get; set; }
        public double Price { get; set; }
        public double AdvancePayment { get; set; }
        public IFormFile ImageFile { get; set; } // For receiving file from client
        public string Image { get; set; } // For storing image path
        public int UserId { get; set; }
        public int CategoryId { get; set; }
        public int ProductId { get; internal set; }
        public int IsActive { get; set; } = 1;
    }
}

File: Quickrent Java/src/main/java/com/quickrent/dto/SellerProductAddDTO.java
package com.quickrent.dto;

import org.springframework.web.multipart.MultipartFile;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
public class SellerProductAddDTO {
    private String title ;
    private String brandName ;
    private String modelName ;
    private String description ;
    private String specifications ;
    private double price ;
    private double advancePayment ;
    private MultipartFile imageFile; // For receiving file from client
    private String image ; // For storing image path
    private int userId ;
    private int categoryId;
    //private int productId = 0;
    private int isActive = 1;
    private int isApproved = 0;
}
